main/glossary/README.md

```diff
@@ -354,6 +350,18 @@ the user either gets what they said they wanted, or gets back (gets a refund) wh
 escrowed. One reason this is possible is if a [proposal](#proposal) doesn't match what the contract expects to do, it
 can immediately cause the [seat](#seat) to exit, getting back the amount it offered.
 
+
+## Passable
+
+A *passable* is something that can be marshalled.
+
+There are three kinds of passables:
+   * Remotables, objects with methods that can be called remotely using `E()`, and their remote Presence.
+   * Pass-by-copy data, such as numbers or hardened records.
+   * Promises for passables.
+
+For more information, see the [Remotable and passable objects documentation](/guides/js-programming/far.md#remotable-and-passable-objects).
```

Comment:
Link seems broken, at least when I try to follow it in github preview. In any case, I see the intended target below. So if this is the correct way to link to it, no change suggested.

Reply:
Links get changed when Vuepress processes the Markdown into HTML for display, at the very least from .md to .html.

Reply:
Added a "Remotable" entry.